apis/policies/v1/policyserver_types.go

```diff
@@ -21,6 +21,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// Security configuration to be used in the Policy Server workload
```

Comment:
Nit:

```suggestion
// PolicyServerSecurity defines securityContext configuration to be used in the Policy Server workload
```

(godoc style)